src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelectorOption.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import TextOverflow from 'app/components/textOverflow';
+import InlineSvg from 'app/components/inlineSvg';
+import {EntryTypeData} from 'app/types';
+
+interface Props {
+  id: string;
+  name?: string;
+  crashed?: boolean;
+  details: ThreadInfo;
+  crashedInfo?: EntryTypeData;
+}
+
+interface ThreadInfo {
+  label: string;
+  filename?: string;
+}
+
+const ThreadsSelectorOption: React.FC<Props> = ({
+  id,
+  name,
+  details,
+  crashed,
+  crashedInfo,
+}) => (
+  <StyledContainer>
```

Comment:
Hmm we might want to call these "wrappers" instead when it's used for visual styles as we sometimes use `Container` for React components that interact with stores.